test/Project.toml

```diff
@@ -1,6 +1,7 @@
 [deps]
 AbstractMCMC = "80f14c24-f653-4e6a-9b94-39d6b0f70001"
 AdvancedHMC = "0bf59076-c3b1-5ca4-86bd-e02cd72cde3d"
+ChainRules = "082447d4-558c-5d27-93f4-14fc19e9eca2"
```

Comment:
Since ChainRules is a dependency of Zygote (and possibly other AD backends) it will always be loaded anyway. So it's not necessary to add it explicitly here, but it might be "less magical".